Artesian/Artesian.SDK.Tests/MarketAssessmentQueries.cs

```diff
@@ -421,6 +421,133 @@ public void MasWithHeaders_FilterId()
         }
         #endregion
 
+        #region Filler
+        [Test]
+        public void FillerDefaultMasInAbsoluteDateRangeExtractionWindow()
+        {
+            using (var httpTest = new HttpTest())
+            {
+                var qs = new QueryService(_cfg);
+
+                var mas = qs.CreateMarketAssessment()
+                       .ForMarketData(new int[] { 100000001 })
+                       .ForProducts(new string[] { "M+1", "GY+1" })
+                       .InAbsoluteDateRange(new LocalDate(2018, 1, 1), new LocalDate(2018, 1, 10))
+                       .ExecuteAsync().Result;
+
+                httpTest.ShouldHaveCalled($"{_cfg.BaseAddress}query/v1.0/mas/2018-01-01/2018-01-10"
+                    .SetQueryParam("id", 100000001)
+                    .SetQueryParam("p", new string[] { "M+1", "GY+1" })
+                    .SetQueryParam("fillerK", FillerKind.LatestValidValue)
```

Comment:
same as per CreateActual, the Default is implemented server-side, not client side.